README.md

```
@@ -46,99 +46,106 @@
+### Implementation Guide
 
-```
+1. Import translation functions:
+```typescript
 import { i18n } from '~core/localization';
+// or for components with HTML
+import { Trans } from 'react-i18next';
 ```
 
-or
+2. Use in code:
```

Markdownlint

Expected: 1; Actual: 2; Style: 1/1/1
Ordered list item prefix

(MD029, ol-prefix)